x-pack/plugins/observability/public/pages/rules/rules.tsx

```diff
@@ -136,13 +161,3 @@ function RulesPage() {
     </ObservabilityPageTemplate>
   );
 }
-
-function WrappedRulesPage() {
-  return (
-    <Provider value={rulesPageStateContainer}>
```

Comment:
Replaced state container with a simpler pattern that offers the same functionality

Reply:
What pattern did you use instead and where is the implementation?

Reply:
I've tagged you in 2 comments.